From 651e635b55479d35d1dd125fa3927f2a42509336 Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:54:31 +0900
Subject: [PATCH 14/23] Revert "Revert "arm64: dts: r8a7795: Add MFIS Lock
 device node""

This reverts commit f5c963caa6ed ("Revert "arm64: dts: r8a7795:
Add MFIS Lock device node"")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index fc1cd08..7a6dc8d 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -761,6 +761,13 @@
 			status = "okay";
 		};
 
+		mfis_lock: mfis-lock@e62600c0 {
+			compatible =    "renesas,mfis-lock-r8a7796",
+					"renesas,mfis-lock";
+			reg = <0 0xe62600c0 0 0x0020>;
+			status = "okay";
+		};
+
 		pfc: pin-controller@e6060000 {
 			compatible = "renesas,pfc-r8a7796";
 			reg = <0 0xe6060000 0 0x50c>;
-- 
1.9.1

